eva/executor/load_executor.py

```diff
@@ -37,6 +37,12 @@ def exec(self, *args, **kwargs):
         elif self.node.file_options["file_format"] == FileFormatType.CSV:
             executor = LoadCSVExecutor(self.node)
 
+        elif self.node.file_options["file_format"] == FileFormatType.PDF:
+            """
+                Executor for extractor from pdf
+            """
+            pass
```

Comment:
So there isn't an executor for `PDF`?

Reply:
Yes, as of now it isn't been added. It is in-progress of development